### ISSUE TITLE
osnet with triplet loss

### ISSUE THREAD

**Issue by machlea** [KaiyangZhou/deep-person-reid#457]
loss:
  name: 'triplet'
  softmax:
    label_smooth: True

train:
  optim: 'amsgrad'
  lr: 0.0015
  max_epoch: 250
  batch_size: 64
  fixbase_epoch: 0
  open_layers: ['classifier']
  lr_scheduler: 'cosine'

    cfg.loss.triplet.weight_t = 0.5 # weight to balance hard triplet loss
    cfg.loss.triplet.weight_x = 1. # weight to balance cross entropy loss

This is my hyperparameters.i even tried loss.triplet.weight_t = 1.0.However, i tried several times, the result cannot be same as paper
mAP:83.0%, rank-1:93.6%
mAP:83.2%, rank-1:93.4%

maybe there are some important hyperparameters that i have ignored?

**Comment by shayan-aqabarary** [comment]
> loss: name: 'triplet' softmax: label_smooth: True
> 
> train: optim: 'amsgrad' lr: 0.0015 max_epoch: 250 batch_size: 64 fixbase_epoch: 0 open_layers: ['classifier'] lr_scheduler: 'cosine'
> 
> ```
> cfg.loss.triplet.weight_t = 0.5 # weight to balance hard triplet loss
> cfg.loss.triplet.weight_x = 1. # weight to balance cross entropy loss
> ```
> 
> This is my hyperparameters.i even tried loss.triplet.weight_t = 1.0.However, i tried several times, the result cannot be same as paper mAP:83.0%, rank-1:93.6% mAP:83.2%, rank-1:93.4%
> 
> maybe there are some important hyperparameters that i have ignored?

I have this problem too, My accuracy for all sections are different with article tables, is it normal?
I want optimize this program, do you have any idea for increase the accuracy?
